text/profiles/0212-profiling-vision.md

```diff
@@ -0,0 +1,173 @@
+
+We will design a profiling data model that will aim to represent the vast
+majority of profiling data with the following goals in mind:
+
+- Profiling formats should be as compact as possible
+- Profiling data should be transferred as efficiently as possible and the model
+  should be lossless with intentional bias for enabling efficient marshaling,
+  transcoding (to and from other formats), and analysis
+- Profiling formats should be able to be unambiguously mapped to the
+  standardized data model (i.e. collapsed, pprof, JFR, etc.)
+- Profiling formats should contain mechanisms for representing relationships
+  between other telemetry signals (i.e. linking call stacks with spans)
+
+## Supporting Legacy profiling formats
+
+For existing profilers we will provide instructions on how these legacy formats
+can emit profiles in a manner that makes them compatible with OpenTelemetry’s
+approach and enables telemetry data correlation.
+
+Particularly for popular profilers such as the ones native to Golang and Java
+(JFR) we will help to have them produce OpenTelemetry-compatible profiles with
+minimal overhead.
+
+## Performance considerations
+
+Profiling agents can be architected in a variety of differing ways, with
+reasonable trade offs made that may impact performance, completeness, accuracy
+and so on. Similarly, the manner in which such a profiler might produce or
+consume OpenTelemetry-compatible data could vary significantly. As such, in our
+standardization effort it is not feasible to be proscriptive on the matter of
+resource usage for profilers.
+
+However, the output of OpenTelemetry's standardization effort must take into
+account that some existing profilers are designed to be low overhead and high
+performance. For example, they may operate in a whole-datacenter, always-on
+manner, and/or in environments where they must guarantee low CPU/RAM/network
+usage. The OpenTelemetry standardisation effort should take this into account
+and strive to produce a format that is usable by profilers of this nature
+without sacrificing their guarantees.
+
+Similar to other Otel signals, we target production environments. Thus, the
+profiling signal must be implementable with low overhead and conforming to
+Otel-wide runtime overhead / intrusiveness and wire data size requirements.
+
+## Promoting Cloud-Native best practices with Profiling
+
+The CNCF’s mission states: _Cloud native technologies empower organizations to
+build and run scalable applications in modern, dynamic environments such as
+public, private, and hybrid clouds_
+
+We will have best-in-class support for profiles emitted in cloud native
+environments (e.g. Kubernetes, serverless, etc), including legacy applications
+running in those environments. As we aim to achieve this goal we will center our
+efforts around making profiling applications resilient, manageable and
+observable.  This is in line with the CNCF and OTEL missions and will thus allow
```

Comment:
Tiny nit: This document goes has both "Otel" and "OTEL" as abbreviations for OpenTelemetry. It should probably be defined as an abbreviation early on, and then stick to one casing.

Reply:
better yet, search/replace OTEL->OpenTelemetry. Abbreviations are good for twitter and slack, in the official documents like OTEPs a full name is preferred.